app/models/api/v1/query_param.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+# To be included in api controllers for handeling query params
```

Comment:
A module to be included in a controller is usually called a *concern*. Have a look at this one:
https://github.com/openfoodfoundation/openfoodnetwork/blob/e066e5ca74d8e6ffe626972cba2d4b32377df587/app/controllers/concerns/raising_parameters.rb#L1-L23

Reply:
Fixed